### PR TITLE
hash alg identifiers and MTI hash alg 

### PR DESCRIPTION
**Pull request by Sakurann**
Addressing comments by introducing 
- hash alg identifier following the prefix
- mandatory to implement hash
- reference to a registry with hash alg identifiers

### CODE REVIEW

File: draft-ietf-oauth-jwk-thumbprint-uri.xml [hunk 1]
```diff
@@ -102,10 +102,21 @@
 	</list>
       </t>
       <t>
-	The prefix is followed by a colon and a JWK Thumbprint value to form
-	a URI representing a JWK Thumbprint.
+	The prefix is followed by a hash algorithm identifier and a JWK Thumbprint value, 
+  separated by the colon to form a URI representing a JWK Thumbprint.
```
Comment (selfissued) [inline draft-ietf-oauth-jwk-thumbprint-uri.xml:106]: separated by the colon -> each separated by colon characters,

File: draft-ietf-oauth-jwk-thumbprint-uri.xml [hunk 2]
```diff
@@ -102,10 +102,21 @@
...
       </t>
 
+      <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
+  <t>
+  Hash algorithm identifier used in JWK Thumbprint URIs are strings
```
Comment (selfissued) [inline draft-ietf-oauth-jwk-thumbprint-uri.xml:111]: identifier -> identifiers

File: draft-ietf-oauth-jwk-thumbprint-uri.xml [hunk 3]
```diff
@@ -102,10 +102,21 @@
...
 
+      <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
+  <t>
+  Hash algorithm identifier used in JWK Thumbprint URIs are strings 
+  registered in the IANA "Named Information Hash Algorithm Registry" <xref target="IANA.Hash.Algorithms"/>.
+  </t>
+
+  <section anchor="MTI" title="Mandatory to Implement Hash Algorithm">
+	<t>
+	  Mandatory to implement Hash Algorithm is SHA-256.
```
Comment (selfissued) [inline draft-ietf-oauth-jwk-thumbprint-uri.xml:117]: How about this?  "To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement."  That way we're saying why there's an MTI algorithm.

File: draft-ietf-oauth-jwk-thumbprint-uri.xml [hunk 4]
```diff
@@ -102,10 +102,21 @@
...
+	</t>
```
Comment (selfissued) [inline draft-ietf-oauth-jwk-thumbprint-uri.xml:118]: I haven't tried compiling it yet, but I think you're missing </section> after the </t>.

**Review by selfissued — CHANGES_REQUESTED** [review]
See the individual comments.

**Comment by selfissued** [comment]
You should also add a bullet describing the change in an -01 history entry.

File: draft-ietf-oauth-jwk-thumbprint-uri.xml (hunk 4 above)
Comment (Sakurann) [reply draft-ietf-oauth-jwk-thumbprint-uri.xml:118]: sorry, what am I missing after what..?

Comment (Sakurann) [reply draft-ietf-oauth-jwk-thumbprint-uri.xml:118]: </section>?

Comment (selfissued) [reply draft-ietf-oauth-jwk-thumbprint-uri.xml:118]: The `<section anchor="MTI" title="Mandatory to Implement Hash Algorithm">` opens a section, which needs to be closed with `</section>` - which would follow the `</t>`.

**Review by selfissued — APPROVED** [review]

**Pull request merged**